local/install_deps_linux.bash

```diff
@@ -65,19 +65,19 @@ if [ "$distro_codename" != "rodete" ]; then
   curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | \
       sudo apt-key add -
 
-  sudo add-apt-repository -y ppa:webupd8team/java
-  sudo apt-get update
 fi
 
 # Install JDK, prerequisite for bazel.
 if [ "$distro_codename" == "trusty" ]; then
+  sudo add-apt-repository -y ppa:webupd8team/java
+  sudo apt-get update
   sudo apt-get install -y oracle-java8-installer
 else
+  sudo apt-get update
   sudo apt-get install -y openjdk-8-jdk
 fi
 
 # Install apt-get packages.
-sudo apt-get update
```

Comment:
Don't like this removal since in future code fixes, people can just remove block above and forget.